scripts/src/utils/models/script_parameters.py

```diff
@@ -0,0 +1,48 @@
+from argparse import Namespace
+from dataclasses import dataclass
+
+from src.utils.file_utils import get_absolute_path
+
+
+@dataclass
+class BaseRunnerParameters:
```

Comment:
Should classes from this file be marked as frozen?

Reply:
Sure, that makes sense, thanks!